libraries/SITL/SIM_Vicon.h

```diff
@@ -75,6 +76,10 @@ class Vicon : public SerialDevice {
 
     void maybe_send_heartbeat();
     uint32_t last_heartbeat_ms;
+
+    uint64_t _pos_delta_prev_time_us;
+    Quaternion _pos_delta_prev_quat;    // Rotation to Frame_aerobody_previous from Frame_earth_NED
```

Comment:
I don't really understand this comment.  "Frame_aerobody_previous" and "Frame_earth_NED" don't appear anywhere except in comments.  I suspect these words come from an out-of-date version of the code.

Reply:
Sorry, I was trying to be explicit about what frames the rotations represent. In this case it is a rotation to previous BODY_FRD from LOCAL_NED. I can be less explicit if you prefer? 
Part of the reason for being explicit was it took me longer than I care to admit to fix this because in SITL the attitude is stored from BODY_FRD to LOCAL_NED which was an unexpected convention. 

Reply:
I think you've fixed this up now so marking as resolved.